### PR TITLE
Feat/cmc 1919 add respondent2 lit friend

### PR DESCRIPTION
**Pull request by Gareth40343**
https://tools.hmcts.net/jira/browse/CMC-1919

Added support for notifying respondent 2 solicitor when litigation friend.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: ccd-definition/AuthorisationCaseField/AuthorisationCaseField.json [hunk 1]
```diff
@@ -2023,6 +2023,7 @@
       {
         "UserRoles": [
           "[APPLICANTSOLICITORONE]",
+          "[RESPONDENTSOLICITORTWO]",
```
Comment (sherlynkhaw) [inline ccd-definition/AuthorisationCaseField/AuthorisationCaseField.json:2026]: Why should solicitor2 have access to respondent1LitigationFriend?

File: ccd-definition/AuthorisationCaseField/AuthorisationCaseField.json [hunk 2]
```diff
@@ -2048,7 +2049,8 @@
       },
       {
         "UserRoles": [
-          "[RESPONDENTSOLICITORONE]"
+          "[RESPONDENTSOLICITORONE]",
```
Comment (sherlynkhaw) [inline ccd-definition/AuthorisationCaseField/AuthorisationCaseField.json:2052]: Does solicitor1 need access to respondent2LitigationFriend

File: ccd-definition/AuthorisationCaseField/AuthorisationCaseField.json (hunk 1 above)
Comment (Gareth40343) [reply ccd-definition/AuthorisationCaseField/AuthorisationCaseField.json:2026]: Respondent2 needs be able to see the respondent1 lit details in the claim details screen. So I gave them read access.

File: ccd-definition/AuthorisationCaseField/AuthorisationCaseField.json (hunk 2 above)
Comment (Gareth40343) [reply ccd-definition/AuthorisationCaseField/AuthorisationCaseField.json:2052]: During a 1v2 same solicitor scenario the solicitor can enter in litigation details for either respondent1, respondent2 or both. So they need write access to respondent2 in that situation.

**Review by HarryH96 — APPROVED** [review]

**Review by sherlynkhaw — APPROVED** [review]

**Review by bhssamray — APPROVED** [review]

**Pull request merged**